Userland/Utilities/mount.cpp

```diff
@@ -197,19 +199,60 @@ static ErrorOr<void> print_mounts()
     return {};
 }
 
+static ErrorOr<void> apply_file_system_specific_options(int mount_fd, Vector<String> const& filesystem_specific_options)
+{
+    for (auto& option : filesystem_specific_options) {
+        auto key_value_option = TRY(option.split_limit('=', 2));
+        MountSpecificFlag flag;
+        flag.key_string_length = key_value_option[0].bytes().size();
+        flag.key_string_addr = key_value_option[0].bytes().data();
+        if (key_value_option.size() > 1 && !key_value_option[1].is_empty()) {
+            auto possible_int_value = AK::StringUtils::convert_to_int<i64>(key_value_option[1]);
+            auto possible_uint_value = AK::StringUtils::convert_to_uint<u64>(key_value_option[1]);
+            if (possible_uint_value.has_value() && possible_int_value.has_value()) {
```

Comment:
Do we need to check both here? `possible_uint_value.has_value()` should be enough? Or are you trying to prevent values that are higher than I64_MAX?

Reply:
I don't really remember the reasoning of this. Possibly it's a way to prevent values higher than I64_MAX, but that seems not reasonable, so I will change this.